scripts/deploy-docs.sh

```diff
@@ -4,20 +4,12 @@ set -e
 
 TARGET_DIR='gh-pages'
 TARGET_BRANCH='master'
-TARGET_REPO_URL="https://$GITHUB_AUTH_TOKEN@github.com/chartjs/chartjs.github.io.git"
+TARGET_REPO_URL="https://$GH_AUTH_TOKEN@github.com/chartjs/chartjs.github.io.git"
```

Comment:
We might be able to use the default GitHub token since the org is the same, but I don't know how to test that. I could not upload the secret with the same name, since GitHub seems to block secrets that have "GITHUB" in them. I renamed to GH and it accepted it.

Reply:
I think its `GITHUB_TOKEN` that is provided automatically by actions. nit.